### PR TITLE
Story Editor: Added Section Header Text Sets

### PR DESCRIPTION
**Pull request by maxyinger**
## Summary
Adds text sets under `Section Header` category in this design file:
https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=837%3A3585

## User-facing changes
NA

## Testing Instructions
Enable the text sets flag in experiments and see the new text sets in the story editor.

---

Partial for #4078

### CODE REVIEW

File: assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json [hunk 1]
```diff
@@ -0,0 +1,2508 @@
...
+          "opacity": 100,
+          "flip": {
+            "vertical": false,
+            "horizontal": false
+          },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "backgroundTextMode": "NONE",
+          "font": {
+            "family": "Poppins",
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json]: I believe this font should be Merriweather not Poppins
(figma)
Screen Shot 2020-09-18 at 9 42 30 AM

(editor)
Screen Shot 2020-09-18 at 9 42 44 AM

File: assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json [hunk 2]
```diff
@@ -0,0 +1,2508 @@
...
+          "opacity": 100,
+          "flip": {
+            "vertical": false,
+            "horizontal": false
+          },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "backgroundTextMode": "NONE",
+          "font": {
+            "family": "Nunito Sans",
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json]: And this is just "Nunito" not "Sans"

(figma)
Screen Shot 2020-09-18 at 9 46 46 AM

(editor)
Screen Shot 2020-09-18 at 9 46 36 AM

**Review by BrittanyIRL — APPROVED** [review]
Awesome! I see 8 new text sets. 

Looks great. I saw two fonts that I think are slightly off but overall this looks great and works well for me in the editor.
Screen Shot 2020-09-18 at 9 51 30 AM

**Review by dmmulroy — APPROVED** [review]
Agree / @BrittanyIRL on the differences but otherwise lgtm

**Review by miina — CHANGES_REQUESTED** [review]
Same here as well as with other sets -- otherwise looks good but the margin between the title and the paragraph varies among the sets whereas in Figma it's fixed 16px

- There are also some minor horizontal alignment issues -- in Figma the sets have 40px from both sides, in the editor it varies a bit, for example:
Screenshot 2020-09-21 at 11 31 11

**Comment by maxyinger** [comment]
> Same here as well as with other sets -- otherwise looks good but the margin between the title and the paragraph varies among the sets whereas in Figma it's fixed 16px
> 
> * There are also some minor horizontal alignment issues -- in Figma the sets have 40px from both sides, in the editor it varies a bit, for example:
> 
> Screenshot 2020-09-21 at 11 31 11

yea spacing was a bit hard. I went off the figma file I was referencing and put everything at the x & y coordinate it specified then once the entire group was positioned, dragged the entire group to proper position in the story. I also think our editor positions bounding boxes around text slightly different than figma, so if something looked off I just tried to manually adjust it as best I could.

If there's a better method I'd be happy to do it but this is the best I could come up with. 

Something to note with going 16px from the bottom of the text bounding box is the difference in bottom for figma vs our app.

**our app**
93746632-be826f80-fbf5-11ea-90f8-69aedbc391a0

**Figma**
Screen Shot 2020-09-23 at 2 57 42 PM
 
this makes 16px from the bottom of the bounding box visually different between figma and the editor. I thoughts tops were more reliable, but still needed some manual approximation to acct for difference imo

**Comment by miina** [comment]
@littlemilkstudio One option could be to create a shape of 16px which could be used as a reference to measure the distance of each set and then removed once it's positioned. WDYT -- would that work / be too complicated?

**Comment by maxyinger** [comment]
@miina yea that's not a bad approach, but wouldn't it still involve some approximation? because there's still space between the bottom of the text and the bottom of the bounding box in figma that we can't get an exact value for and we'd need to line the 16px shape to the bottom of where the figma bounding box _would_ be?

Or am I misunderstanding the suggestion?

**Comment by miina** [comment]
> because there's still space between the bottom of the text and the bottom of the bounding box in figma that we can't get an exact value for and we'd need to line the 16px shape to the bottom of where the figma bounding box would be?

Yep, true, it still wouldn't be precise but it might give a bit better reference.

Anyway, if you feel like that's OK as it is then let's just merge and Sam can do the UAT later.

**Review by miina — APPROVED** [review]
LGTM -- let's follow up with adjustments after UAT if needed.

**Pull request merged**